src/policy.rs

```diff
@@ -139,16 +140,7 @@ impl Policy {
                         }
                         // TODO(eastizle): not all fields are strings
                         // https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes
-                        Some(attribute_bytes) => match String::from_utf8(attribute_bytes) {
-                            Err(e) => {
-                                debug!(
-                                    "#{} build_single_descriptor: failed to parse selector value: {}, error: {}",
-                                    filter.context_id, attribute_path, e
-                                );
-                                return None;
-                            }
-                            Ok(attribute_value) => attribute_value,
-                        },
+                        Some(attribute_bytes) => Attribute::parse(attribute_bytes).ok()?,
```

Comment:
I'm not 100% here, because this could take advantage of the `type_of` to determine the type instead of assuming it's a `String`